txtdirect.go

```diff
@@ -173,12 +180,31 @@ func Redirect(w http.ResponseWriter, r *http.Request, c Config) error {
 	}
 
 	rec, err := getRecord(host, c, w, r)
-	r = rec.addToContext(r)
 	if err != nil {
 		fallback(w, r, "global", http.StatusFound, c)
 		return nil
 	}
 
+	// Add the upstream zone address from the use= fields to the request context
+	// and replace the source record with the upstream record
```

Comment:
We don't replace the source record with the upstream record anymore or?

Reply:
Removed the second line. Leftover from the previous implementation.